src/test/java/io/airlift/units/TestDataSizeValidator.java

```diff
@@ -110,6 +138,13 @@ public void testFailsMaxDataSizeConstraint()
                 .allMatch(MaxDataSize.class::isInstance);
         assertThat(violations.stream().map(violation -> violation.getPropertyPath().toString()))
                 .containsOnly("constrainedByMax", "constrainedByMinAndMax");
+
+        violations = VALIDATOR.validate(new ConstrainedOptionalDataSize(Optional.of(new DataSize(11, MEGABYTE))));
+        assertThat(violations).hasSize(2);
+        assertThat(violations.stream().map(violation -> violation.getConstraintDescriptor().getAnnotation()))
```

Comment:
Use `extracting` and `allSatisfy`

Reply:
I don't see `. allSatisfy`. The `.allMatch` here is assertj's not Stream's.